hook/ip-dest-src-module.c

```diff
@@ -0,0 +1,54 @@
+
+#include <linux/init.h>
+#include <linux/module.h>
+#include <linux/kernel.h>
+#include <linux/netfilter.h>
+#include <linux/netfilter_ipv4.h>
+#include <linux/ip.h>
+
+MODULE_LICENSE("GPL");
+MODULE_AUTHOR("Eitan");
```

Comment:
I hope those kernel modules are working well if you are using my name for Author ;)

Reply:
Fixed it.